<launch>

  <include file="$(find openni_launch)/launch/openni.launch">
  </include>

  <node name="serial_node" pkg="rosserial_python" type="serial_node.py"/>
  <node pkg="explorer" name="odometry" type="odometry"/>
  <node pkg="explorer" name="motors" type="motors"/>
  <node pkg="explorer" name="EKF" type="EKF"/>
  <node pkg="explorer" name="wall_follower" type="wall_follower" output="screen"/>
  <node pkg="explorer" name="phidgets" type="phidgets" output="screen"/>

</launch>
